libs/server/Resp/RespServerSessionSlotVerify.cs

```diff
@@ -65,9 +55,10 @@ bool CanServeSlot(RespCommand cmd)
                             csvi.step = findRange.KeyStep;
                             break;
                         case FindKeysKeyNum:
+                            var beginSearch = (BeginSearchIndex)specs[0].BeginSearch;
```

Comment:
Why is there a beginSearch and a searchIndex above?

Reply:
Removed this because it is not needed. Forgot that searchIndex was available.